packages/react-email/source/commands/dev.ts

```diff
@@ -35,8 +36,11 @@ export const dev = async ({ dir, port }: Args) => {
   const watcherInstance = createWatcherInstance(emailDir);
   try {
     const hasReactEmailDirectory = checkDirectoryExist(REACT_EMAIL_ROOT);
+    const cwd = await findRoot(CURRENT_PATH)
+      .then((v) => v.rootDir)
+      .catch(() => CURRENT_PATH);
```

Comment:
Should use `await` or `then` here, not both right?

Reply:
To be the promise chainable and catch if it throws, I think this is the cleanest way.
Without the .then should be:

```ts
    let cwd: string;
    try {
      cwd = (await findRoot(CURRENT_PATH)).rootDir;
    } catch (error) {
      cwd = CURRENT_PATH;
    }
```

If we do this:
```ts
    const cwd = (await findRoot(CURRENT_PATH)).rootDir
      .catch(() => CURRENT_PATH);
```
.catch is not chainable anymore `Property 'catch' does not exist on type 'string'. Did you mean 'match'?ts(2551)`

Reply:
Yup, if we're going if `then` then we don't need the `await`

Reply:
ah right

Reply:
Let me know when you remove, and we can merge it.